.github/workflows/cherry-pick-since-release-branch.yml

```diff
@@ -23,7 +23,12 @@ jobs:
       - name: Get all release branches including label version and higher
         id: filter-release-branches
         run: |
-          label="${{ github.event.label.name }}"
+          if [[ ${{ github.event.action }} == 'labeled' ]]; then
+            label="${{ github.event.label.name }}"
+          else
+            labels='${{ toJson(github.event.pull_request.labels) }}'
+            label=$(echo "$labels" | jq -r '.[] | select(.name | contains("need-cherry-pick-since")).name' | sort -V | head -n 1)
+          fi
```

Comment:
Similarly on the single cherry-pick action:
https://github.com/risingwavelabs/risingwave/blob/5a06d47cf43d605e92ce5c46960e17ae671cdf4c/.github/workflows/cherry-pick-to-release-branch.yml#L21-L26